transforms/language/doc_quality/python/README.md

```diff
@@ -0,0 +1,76 @@
+# Document Quality Transform 
+Please see the set of
+[transform project conventions](../../../README.md#transform-project-conventions)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary 
+This transform will calculate several metrics related to document, which are usuful to see the quality of document. 
+
+In this transform, following metrics will be included:
+- Gopher (Deepmind): filter docs that
+  - do not contain between 50 and 100,000 words
+  - mean word length is outside the range of 3 to 10 characters;
+  - symbol-to-word ratio > 0.1 for either the hash symbol or the ellipsis;
+  - > 90% of lines starting with a bullet point,
+  - > 30% ending with an ellipsis.
+  - Require that 80% of words in a document contain at least one alphabetic character, and apply a "stop word" filter, to remove documents that do NOT contain at least TWO of the following English words: the, be, to, of, and, that, have, with; this adequately deals with ostensibly English documents that contain no coherent English text.
+- Perplexity score (KenLM+sentencepiece) suggested in Gopher The smaller the perplexity score, the closer is the text to the targeted domain (i.e., en Wikipedia). Journalistic and well written content. Distribution of perplexity for different languages may have different shapes.
+
+
+
+## Configuration and command line Options
+
+The set of dictionary keys holding [DocQualityTransform](src/doc_quality_transform.py) 
+configuration for values are as follows:
+
+* _docq_text_lang_ - specifies language used in the text content. By defaut, "en" is used.
```

Comment:
I think your configuration keys do not include the `docq_` prefix.

Reply:
Ah, yes. 
I just followed the patter in noop. https://github.com/IBM/data-prep-kit/blob/dev/transforms/universal/noop/python/README.md#configuration-and-command-line-options
I'll update to remove `docq_`.